deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -301,6 +301,19 @@ def deploy_seqr(settings):
     deploy_pod("seqr", settings, wait_until_pod_is_ready=True)
 
 
+def redeploy_seqr(deployment_target):
+    print_separator('re-deploying seqr')
+
+    seqr_pod_name = get_pod_name('seqr', deployment_target=deployment_target)
+    if not seqr_pod_name:
+        raise ValueError('No seqr pod found, unable to re-deploy')
+    sleep_until_pod_is_running('seqr', deployment_target=deployment_target)
+
+    run_in_pod(seqr_pod_name, 'git pull', verbose=True)
+    run_in_pod(seqr_pod_name, './manage.py migrate', verbose=True)
+    run_in_pod(seqr_pod_name, '/usr/local/bin/restart_server.sh')
```

Comment:
This is the series of commands I was running when I ssh'd in and did the in-place deployment. It seems safer to have that as a script that runs, rather than me copy-pasting commands in